deno_typescript/lib.rs

```diff
@@ -204,6 +204,10 @@ pub fn mksnapshot_bundle_ts(
   state: Arc<Mutex<TSState>>,
 ) -> Result<(), ErrBox> {
   let runtime_isolate = &mut Isolate::new(StartupData::None, true);
+  runtime_isolate.register_op(
+    "fetch_asset",
```

Comment:
It may be worth adding a comment there that this op is added just to handle hydration of `oldProgram` - ideally we'll use `CompilerIsolate` here (after landing #3116)

Reply:
actually, I realised we don't need this OP as part of the cli when we do it this way, I can strip that out... because everything gets parsed and becomes part of the snapshot, basically the op never fires and doesn't need to be registered in the compiler isolate.